odoo/tools/mail.py

```diff
@@ -635,3 +636,15 @@ def encapsulate_email(old_email, new_email):
         name_part,
         new_email_split[0][1],
     ))
+
+
+def url_domain_extract(url):
+    """ Extract the company domain to be used by IAP services notably. Domain
+    is extracted from an URL e.g:
+        - www.info.proximus.be -> proximus.be
+    """
+    parser_results = werkzeug.urls.url_parse(url)
+    company_hostname = parser_results.netloc
```

Comment:
@tde-banana-odoo if we want to use werkzeug instead of urllib, then we have to slightly change the backporting, is that okay ? (change parser_results and .hostname becomes .netloc

Reply:
Seems ok. In forward port we will update existing code to use werkzeug also.